src/main/java/org/generationcp/middleware/api/germplasm/GermplasmServiceImpl.java

```diff
@@ -129,6 +128,9 @@
 	@Autowired
 	private GermplasmAttributeService germplasmAttributeService;
 
+	@Autowired
```

Comment:
have you checked if the bean is created in fieldbook and workbench? 
it will try to autowire germplasmNameTypeService and I think it is not declared in Fieldbook and Workbench 
@cuenyad